tracer/test/test-applications/integrations/LogsInjection.NLogFallback.VersionConflict.2x/Program.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using LogsInjectionHelper.VersionConflict;
+using NLog;
+using NLog.Config;
+
+namespace LogsInjection.NLogFallback.VersionConflict_2x
```

Comment:
FYI, there are issues with the NLog testing which I address in the direct log submission PR. I should probably split that out into a separate PR